en/docs/guides/login/request-app-claims.md

```diff
@@ -0,0 +1,61 @@
+# Configure attributes for service providers
+
+This page guides you through configuring the attribute mappings with Identity Server and the application to pass the 
```

Comment:
attribute mappings for WSO2 Identity Server, and the application to pass the configured attribute values to the application.